src/components/App/App.js

```diff
@@ -1,36 +1,45 @@
 import React, { useState, useEffect } from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import ProtectedRoute from '../ProtectedRoute/ProtectedRoute';
 import Header from '../Header/Header';
+
 import About from '../About/About';
 import Footer from '../Footer/Footer';
 import Main from '../Main/Main';
-import SavedNews from '../SavedNews/SavedNews';
+// import SavedNews from '../SavedNews/SavedNews';
 import SavedArticles from '../SavedArticles/SavedArticles';
 import SignInPopup from '../SignInPopup/SignInPopup';
 import SignUpPopup from '../SignUpPopup/SignUpPopup';
 import InfoPopup from '../InfoPopup/InfoPopup';
 import './App.css';
+import NewsApi from '../../utils/NewsApi';
+import MainApi from '../../utils/MainApi';
+import * as auth from '../../utils/auth';
+import { CurrentUserContext } from '../../context/CurrentUserContext';
 
```

Comment:
when I log in my saved articles don't get loaded and there is a server error in the console but when I reload the page the articles appear. (See screen shot http://joxi.ru/BA0YjEvSpgEexr)